### PR TITLE
Missing processing

### PR DESCRIPTION
**Pull request by 14thibea**
Add a new iotool to compute which processing was performed for each subject/session.
Contains t1 and pet pipelines (pet-linear should be added when it is ready).

### CODE REVIEW

File: clinica/iotools/utils/pipeline_handling.py [hunk 1]
```diff
@@ -4,388 +4,274 @@
...
-        atlas_selection = None
 
-    if "group_selection" in kwargs.keys():
-        group_selection = kwargs["group_selection"]
-    else:
-        group_selection = None
+def volume_pipeline(caps_dir, df, pipeline_path, pipeline_name,
+                    group_selection=None, atlas_selection=None,
+                    pvc_restriction=None, tracers_selection=None):
+    """Merge data of the t1-volume pipeline to the merged file containing the BIDS information.
```
Comment (alexandreroutier) [inline clinica/iotools/utils/pipeline_handling.py:125]:
```suggestion
    """Merge data of the t1-volume and pet-volume pipelines to the merged file containing the BIDS information.
```

File: clinica/iotools/utils/pipeline_handling.py [hunk 2]
```diff
@@ -4,388 +4,274 @@
...
+
+    Args:
+        caps_dir: the path to the CAPS directory
+        df: the DataFrame containing the BIDS information
+        group_selection: allows to choose the DARTEL groups to merge. If None all groups are selected.
+        atlas_selection: allows to choose the atlas to merge. If None all atlases are selected.
```
Comment (alexandreroutier) [inline clinica/iotools/utils/pipeline_handling.py:132]: Docstrings for pvc_restriction and tracers_selection are missing.

File: docs/IO.md [hunk 3]
```diff
@@ -106,6 +106,31 @@ If an input list of subjects and sessions is given, the merged file will only ga
     ...
     ```
 
+##`check-missing-processing` - Check missing processing in a CAPS directory
+Starting from a CAPS compliant dataset, this command creates a TSV file with columns
+`participant_id`, `session_id` and names corresponding to steps of `t1-volume`, `t1-freesurfer`, 
+`t1-linear`, `pet-volume` and `pet-surface`. For PET pipelines one column is created per tracer
+and the PVC option is considered for `pet-volume`.
```
Comment (alexandreroutier) [inline docs/IO.md:113]: Could you add in your example below what would look a TSV with 2 tracers?

**Review by alexandreroutier — APPROVED** [review]
It looks great :)

File: clinica/iotools/utils/pipeline_handling.py (hunk 1 above)
Comment (14thibea) [reply clinica/iotools/utils/pipeline_handling.py]: Done in https://github.com/aramis-lab/clinica/pull/166/commits/4522a7d336c36b2b149249362a839d4033ff8517

File: clinica/iotools/utils/pipeline_handling.py (hunk 2 above)
Comment (14thibea) [reply clinica/iotools/utils/pipeline_handling.py]: Done in https://github.com/aramis-lab/clinica/pull/166/commits/4522a7d336c36b2b149249362a839d4033ff8517

**Review by mdiazmel — APPROVED** [review]
HI @14thibea ,
Seems ok to me! I'm going to merge this PR! I think that the better will be to do a "squash and merge" to resume this contribution in a single commit and to avoid merging conflicts with the `dev` branch. Are you ok with this?

**Pull request merged**